js/home.js

```diff
@@ -17,7 +17,81 @@ inputField.addEventListener("keypress", (e) => {
     }
   }
 });
+////////////////////fetch the API of  the city and cuntry
+fetch("https://countriesnow.space/api/v0.1/countries/population/cities")
+    .then(
+        e => e.json()//json the data
+    )
+    .then(data1 => {
+        let arr = []//make the emputy array to push the data to it
+        //for loop to push data
+        for (key of data1.data) {
+            arr.push(key.city)
+            arr.push(key.country)
+        }
+        //to remove the same data 
+        let seto = [...new Set(arr)]
+        //select the element
+        const searchWrapper = document.querySelector(".input-parent");
+        const inputBox = searchWrapper.querySelector("input");
+        const suggBox = document.querySelector(".recommends");
+        //when click in the input call the showList function
+        inputBox.addEventListener("click", showList)
+        //show the autocomplet list 
+        function showList() {
+            let arr = seto.map(qq => {
+                return qq = '<span>' + qq + '</span>'
+            }).join("")
+            suggBox.innerHTML = arr
```

Comment:
don't use `innerHTML` when you can use other safe property 

'innerHTML' Presents a Security Risk